src/common/keyboard.js

```diff
@@ -92,7 +92,7 @@ if (!('key' in window.KeyboardEvent.prototype)) {
   ( window.KeyboardEvent.prototype
   , 'key'
   , { get: getKey
-    /*::, value: void(0)*/
+    , value: void(0)
```

Comment:
@Gozala, well but this solution is definitely the worst of all worlds: `Object.defineProperty` throws if you specify both `value` and either `get` or `set` (or both). While the enum/variant type would be best, the version with all values optional would be better than what they have now.